hr_timesheet_holidays/wizard/holidays_import_view.xml

```diff
@@ -1,37 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-  <data>
+<odoo>
 
     <record id="view_hr_timesheet_holidays_import" model="ir.ui.view">
-      <field name="name">hr.timesheet.holidays.import</field>
-      <field name="model">hr.timesheet.holidays.import</field>
-      <field name="arch" type="xml">
-        <form string="Select holidays to import">
-          <field name="holidays_ids" nolabel="1" />
-          <group colspan="4" col="6">
-            <button icon="gtk-cancel" special="cancel" string="Cancel"/>
-            <button icon="gtk-ok" string="Import" name="import_holidays" type="object"/>
-          </group>
-        </form>
-      </field>
+        <field name="name">hr.timesheet.holidays.import</field>
+        <field name="model">hr.timesheet.holidays.import</field>
+        <field name="arch" type="xml">
+            <form string="Select holidays to import">
+                <field name="holidays_ids" nolabel="1" />
+                <group colspan="4" col="6">
+                    <button special="cancel" string="Cancel"/>
+                    <button string="Import" name="import_holidays" type="object"/>
```

Comment:
You should put those buttons in the footer. See attach screenshot.
![timesheet_holidays01](https://user-images.githubusercontent.com/8035793/31119208-7875cba2-a830-11e7-8e0f-18267ea05775.png)

Reply:
You're right @adrienpeiffer Thanks